script.js

```diff
@@ -1,4 +1,435 @@
+  }
+  if (look == "paper" || look == "reversed paper") {
+    logo = "📜";
+  }
+  if (look == "stone" || look == "reversed stone") {
+    logo = "🗿";
+  }
+  return logo;
+};
+
+// ------------------  Main game ---------------------------------
+//-------------------- Main game ---------------------------------
+
+// define main function to enter different game modes and versions
+var main = function (input) {
+  var myOutputValue = "";
+  console.log(currentGameMode);
+  if (input == "refresh") {
+    currentGameMode = "Choose your game";
+    myOutputValue = `Hello ${userName}! Please choose the game mode you'll like to play. <br><br>Type in "Normal", "Reversed", "Korean", "AI"`;
+  } else if (currentGameMode == "enter user name") {
+    myOutputValue = setUserName(input);
+  } else if (currentGameMode == "Choose your game") {
+    myOutputValue = startGame(input);
+  } else if (currentGameMode == "Normal") {
+    myOutputValue = normalGame(input);
+  } else if (currentGameMode == "Reversed") {
+    myOutputValue = reversedGame(input);
+  } else if (currentGameMode == "Korean") {
+    myOutputValue = koreanGame(input);
+  } else if (currentGameMode == "AI") {
+    myOutputValue = aiGame(input);
+  }
+  // console log to track which game mode we are current in
+  console.log("currentGameMode", currentGameMode);
+  return myOutputValue;
+};
+
+//----------- Set user name game mode --------------------
+
+var setUserName = function (playerName) {
+  userName = playerName;
+  currentGameMode = "Choose your game";
+  var myOutputValue = `Hello ${userName}! Please choose the game mode you'll like to play. <br><br>Type in "Normal", "Reversed", "Korean", "AI"`;
+  return myOutputValue;
+};
+
+// ----------- game mode to choose the type of game---------
+
+var startGame = function (input) {
+  var myOutputValue = `Please type "Normal", "Reversed", "Korean", "AI"`;
+  if (input == "Normal") {
+    currentGameMode = "Normal";
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>This is the normal scissors, paper, stone game.<br><br>Start by typing in "scissors", "paper" or "stone"`;
+  }
+  if (input == "Reversed") {
+    currentGameMode = "Reversed";
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>This is the REVERSED of scissors, paper, stone game.<br><br>Start by typing in "scissors", "paper" or "stone"`;
+  }
+  if (input == "Korean") {
+    currentGameMode = "Korean";
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>The Ultimate Winner is determined in a draw game.<br><br>The Ultimate Winner is the previous winner when the current game is a draw!<br><br>Start by typing in "scissors", "paper" or "stone"`;
+  }
+  if (input == "AI") {
+    currentGameMode = "AI";
+    myOutputValue = `You have selected Computer Vs AI as your game of choice! <br><br>An AI machine will pick for you!<br><br>Start by pressing Submit`;
+  }
+  return myOutputValue;
+};
```

Comment:
Your gaming messages to user is spot on. This is also part of the coding aspect, which some ignore, namely, thinking for the user. 

You can use SCREAM_CASE variables here again for the diff game modes